### ISSUE TITLE
A class for reference type parameter of a method is not created in output. 

### ISSUE THREAD

**Issue by tahiat** [njit-jerse/specimin#47]
The input class is as follow:-
```
package com.example;
class Foo {
    void bar(Baz b) {
        System.out.println("hello");
    }
}
```

SPECIMIN output has the content of Foo class. However no Baz class is created. **Should not Specimin create a Baz.java class also?**

Command used: `./gradlew run --args='--outputDirectory "tempDir" --root "src/test/resources/unusedsecondclass/input/" --targetFile "com/example/Foo.java"  --targetMethod "com.example.Foo#bar(Baz)"'`

**Comment by LoiNguyenCS** [comment]
The good news is that we already have PR #46 that can fix this bug. This PR is almost ready to merge, I think.

**Comment by kelloggm** [comment]
Fixed by #46
